package.xml

```diff
@@ -6,7 +6,7 @@
   <name>libyaml_vendor</name>
   <version>1.2.0</version>
   <description>Vendored version of libyaml.</description>
-  <maintainer email="michel@ekumenlabs.com">Michel Hidalgo</maintainer>
```

Comment:
Let's make sure to automatically convert old maintainers to authors to recognize their contribution.